src/components/Button/Button.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+import { Container } from './Button.styles';
+
+const Button = ({ children, onClick }) => <Container onClick={onClick}>{children}</Container>;
```

Comment:
1. 이 Button 컴포넌트를 submit이 아닌 button type으로 사용하려면 바로 사용하지 못하고 props를 또 추가해줘야할 것 같아요.
2. disabled만 주려고 해도 마찬가지구요

HTML button element의 props를 모두 받을 수 있게 확장해보면 어떨까요?

```suggestion
const Button = props => <Container {...props} />;
```

Reply:
지금 리덕스 비동기 로직에 허덕허덕 대고 있었는데,,,ㅠ ㅠ 알림 메일 받고 깜짝 놀랐네요 호호호호호호호호호호호

Reply:
와 type이나 다른 props에 대한 확장성 고려를 못했네요 ㅎㅎㅎㅎ 피드백 감사합니다 >_<